mne/viz/epochs.py

```diff
@@ -173,6 +174,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     else:
         picks = np.atleast_1d(picks)
 
+    if "invert_y" in ts_args:
+        raise NotImplementedError("'invert_y' found in 'ts_args'. "
+                                  "This currently")
```

Comment:
Is this some sort of existential statement?

Reply:
Damnit! The syntax is still broken!